keel-web/src/main/kotlin/com/netflix/spinnaker/keel/apidocs/TitusClusterSpecSchemaCustomizer.kt

```diff
@@ -0,0 +1,39 @@
+package com.netflix.spinnaker.keel.apidocs
+
+import com.netflix.spinnaker.keel.api.titus.TitusClusterSpec
+import com.netflix.spinnaker.keel.api.titus.TitusServerGroupSpec
+import io.swagger.v3.core.converter.ModelConverterContext
+import io.swagger.v3.oas.models.media.Schema
+import org.springframework.stereotype.Component
+
+@Component
+class TitusClusterSpecSchemaCustomizer : AbstractSchemaCustomizer() {
```

Comment:
_Man_, it's taking a lot of code to generate these API specs/docs... In my mind, part of the goal was always to auto-generate the specs without a lot of additional work. The fact that we have to add so many customizations outside the actual classes feels brittle and error-prone to me. I wonder if we could come up with a different alternative (maybe our own annotations?)...

Reply:
Yeah, I'm actually looking into that today